packages/react-vanilla-components/__tests__/components/FileUpload.test.tsx

```diff
@@ -20,6 +22,9 @@ const field = {
   maxFileSize: "5MB",
   fieldType: "file-input",
   visible: true,
+  properties: {
+    dragDropText: "Drag and drop to Upload"
```

Comment:
It should be "fd:dragDropText"

Reply:
Fetching this from JSON. Correction needs to be done in core component